block_cascade/cli/crud.py

```diff
@@ -0,0 +1,259 @@
+import click
+import json
+from google.cloud import aiplatform_v1beta1 as aiplatform
+from google.protobuf import json_format
+
+from block_cascade import GcpResource
+from block_cascade.config import find_default_configuration
+from block_cascade.consts import SERVICE
+from block_cascade.utils import get_gcloud_config
+
+
+def get_gcp_project(config: dict) -> str:
+    """Return the GCP project from a local config."""
+    return config.get("core", {})["project"]
+
+
+def get_gcp_region(config: dict) -> str:
+    """Return the GCP region from a local config."""
+    return config.get("compute", {})["region"]
+
+
+def get_persistent_resource_payload(resource: GcpResource) -> dict:
+    """Generate a persistent resource payload from a GcpResource."""
+
+    CPU_MACHINE_TYPE = resource.chief.type
```

Comment:
opt: upper cased, snake cased variable names usually indicate a [module level constant](https://peps.python.org/pep-0008/#constants)

Reply:
Yeah I copied this from the GCP notebook and forgot to go back and change them : )